lk2nd/device/dts/msm8916/msm8916-samsung.dts

```diff
@@ -850,4 +850,22 @@
 			i2c-scl-gpios = <&tlmm 3 I2C_GPIO_FLAGS>;
 		};
 	};
+
+	j5xnltegn {
+		model = "Samsung Galaxy J5 2016 (SM-J510GN)";
+		compatible = "samsung,j5xnlte", "samsung,j5x";
+		lk2nd,match-bootloader = "J510GN*";
+
+		lk2nd,dtb-files = "msm8916-samsung-j5x";
+
+		qcom,msm-id = <QCOM_ID_MSM8916 0>;
+		qcom,board-id = <0xCE08FF01 4>;
```

Comment:
This file is sorted by revision, I'd appreciate if you move this node into "rev 4"

Also I'd appreciate if you change the commit message similarly to others changing this file

Reply:
Okay. Sorry, It's my first time.